compiler/AST/expr.cpp

```diff
@@ -97,7 +97,7 @@ static void codegenCall(const char* fnName, GenRet a1, GenRet a2, GenRet a3, Gen
 static void codegenCall(const char* fnName, GenRet a1, GenRet a2, GenRet a3, GenRet a4, GenRet a5);
 
 static GenRet codegenZero();
-static GenRet codegenOne();
+//static GenRet codegenOne();
```

Comment:
I'm not sure, leaving it there causes unused function warnings/errors, but it seems like something that could become useful again in the future. I might leave it for this PR and go back and make a `codegenNumber(int x)` or something along those lines.